e2e/config.js

```diff
@@ -2,7 +2,11 @@
 
 module.exports = {
   // users
-  localAuthorityEmail: 'kurt@swansea.gov.uk',
+  swanseaLocalAuthorityEmailKurt: 'kurt@swansea.gov.uk',
```

Comment:
I would prefer to make these names more generic so that we can always change user names without affecting E2E too much. Just have no good suggestion for the var name. Will came up with something tomorrow.

Reply:
changed whilst removing. gone for swanseaLocalAuthorityUserOne etc